pkg/dazzle/registry.go

```diff
@@ -39,14 +39,29 @@ const (
 	mediaTypeTestResult = "application/vnd.gitpod.dazzle.tests.v1+json"
 )
 
+// Registry provides container registry services
+type Registry interface {
```

Comment:
Nice one. We could consider (in a next step after this PR) to `mock` this interface proper, and make `removeBaseLayer` use this interface, too.

Reply:
> next step 
+1

> mock this interface proper
mock or fake? 

> make removeBaseLayer use this interface
I was going to look at expanding the testing and it would make sense to take a look at doing it then.